src/fusion.cpp

```diff
@@ -108,6 +108,50 @@ miopenStatus_t ConvBiasActivFusion(Handle& handle,
     return miopenStatusSuccess;
 }
 
+static auto MakeConvBiasActivFusionInvokeParams(const FusionContext& context,
```

Comment:
```suggestion
static auto AllocateBuffersAndMakeConvBiasActivFusionInvokeParams(const FusionContext& context,
```
Looks ugly (as it should ;)